src/Hyperparameters.jl

```diff
@@ -103,6 +103,16 @@ function hyperparams(
     return (; parameters...)
 end
 
+"""
+    save_hyperparam(name::Symbol, value, prefix::AbstractString="")
+
+Save value to the enviroment variables and the global `HYPERPARAMETERS` dictionary.
+"""
+function save_hyperparam(name::Symbol, value; prefix::AbstractString="")
+    ENV[uppercase(string(prefix, name))] = string(value)
+    _set_hyperparam(name, value)
```

Comment:
this is not nesc the same effect as putting it into the enviroment variable and then reading it.
SInce it might not parse.
But this is probably the effect that is wanted.